### PR TITLE
testing/amiitool: new aport

### PR DESCRIPTION
**Pull request by TBK**

### CODE REVIEW

File: testing/amiitool/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Contributor: TBK <alpine@jjtc.eu>
+# Maintainer: TBK <alpine@jjtc.eu>
+pkgname=amiitool
+pkgver=r2
```
Comment (Ikke) [inline testing/amiitool/APKBUILD]: Sadly this does not work as expected with apk, as it treats it as a postfix (like a pre-release). So it's best to just use `2` as the version and then prepend the 'r' where necessary.

Comment (TBK) [reply testing/amiitool/APKBUILD]: Roger, I already suggested upstream to switch to semver.

Will fix asap.

Comment (Ikke) [reply testing/amiitool/APKBUILD]: You forgot to put the 'r' back in the url, it now 404's

Comment (TBK) [reply testing/amiitool/APKBUILD]: doh, too many balls in the air. I have made the corrections.

**Pull request closed without merging**